java-ast/addon/src/main/java/org/jboss/windup/ast/java/ReferenceResolvingVisitor.java

```diff
@@ -625,35 +516,14 @@ public boolean visit(FieldDeclaration node)
                     //additionally add Enum Constant type reference
                     if (resolveBinding.isEnum())
                     {
-                        processTypeAsEnum(resolveBinding, ResolutionStatus.RESOLVED, TypeReferenceLocation.ENUM_CONSTANT, lineNumber, columnNumber, node.getLength(), node.toString());
+                        processTypeAsEnum(resolveBinding, expressionName, ResolutionStatus.RESOLVED, lineNumber, columnNumber, node.getLength(), extractDefinitionLine(node.toString()));
                     }
                     
                     processTypeBinding(resolveBinding, ResolutionStatus.RESOLVED, TypeReferenceLocation.VARIABLE_INITIALIZER,
                                 lineNumber,
                                 columnNumber, node.getLength(), node.toString());
                 }
             }
-            else
-            {
-                if (expression != null && ClassNameUtil.isConstant(expression.toString()))
-                {
-                    // if there is static import and constant is not FQCN
-                    String qualifiedConstant = state.getClassNameToFQCN().get(expression.toString());
-                    
-                    if (qualifiedConstant != null)
-                    {
-                        PackageAndClassName packageAndClassName = PackageAndClassName.parseFromQualifiedName(qualifiedConstant);
-    
-                        // the Enum Class and Constant doesn't have to be FQCN as package could miss 
-                        if (packageAndClassName.constantName != null) 
-                        {
-                            //additionally add Enum Constant type reference if found in String
-                            // verify it is ENUM and pass it ??? FIXME: is it really accesible the enum constant here or just ENUM
-                            processStringAsEnum(packageAndClassName, ResolutionStatus.RECOVERED, TypeReferenceLocation.ENUM_CONSTANT, lineNumber, columnNumber, node.getLength(), node.toString());
-                        }
-                    }
-                }
-            }
```

Comment:
JDT deals with that case automatically. The enum itself needs to be on the analysis path (or the additional associated library jars), though in order for jdt to know that it is an enum.